test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] Good job so far 👏. While It looks like the `test.md` file is still visible on Github despite being added in the `.gitignore` file. This may be due to that you have pushed it to GitHub together with other commit before you add it in `.gitignore` file. In order to fix this, **please delete this file manually here on Github and then delete it also locally in your machine. Create a new one and before doing anything else, add its name in the `.gitignore` file and make sure that it is marked as untracked by git.** After all of this, you are sure that it will not be visible on GitHub when you push other codes 😉

Reply:
Hi  @BertrandConxy ! thanks for your review, it was very helpful. I hope this time you find everything right.

Have a nice day.
Lucas